over.css

```diff
@@ -0,0 +1,73 @@
+*{
```

Comment:
I generally try to avoid * selector because it's just too unpredictable. You don't know when the styles you're applying will have unintended effects. Can you put this on body?